constants.py

```diff
@@ -180,12 +180,15 @@ class DiverCarlChute:
 class CaptainPlanetConsts:
     kMotorCanId = 13
     kMotorInverted = False
-    kFrontBreakBeam = 1
-    kBackBreakBeam = 0
-    kDefaultSpeed = 0.15
-    class BreakBeam(Enum):
-        FRONT = 1  # Closest to large green wheels
-        BACK = 2
+    kCurrentLimitAmps = 20
```

Comment:
I think 20 is pretty high for this
```suggestion
    kCurrentLimitAmps = 7
```

Reply:
Jordan found that this seemed to be working